NOnion/Utility/HiddenServicesUtility.fs

```diff
@@ -5,7 +5,7 @@ open System
 open NOnion
 open DateTimeUtils
 
-module HSUtility =
+module HiddenServicesUtility =
```

Comment:
@aarani this kind of change should not be in this commit but in a refactoring commit